pystan/experimental/misc.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import re
+import os
+
+logger = logging.getLogger('pystan')
+
+def fix_include(model_code):
+    """Function to normalize (remove whitespace) around the #include statements.
+    
+    Note
+    ----
+    This functions return edited version of the model_code.
+
+    Parameters
+    ----------
+    model_code, str
+
+    Returns
+    -------
+    new_model_code, str
```

Comment:
Google docstrings are a little different: http://www.sphinx-doc.org/en/stable/ext/example_google.html

```
model_code (str): model code
```

and in returns you don't mention the variable so 

```
  str: reformatted model code
```

Reply:
Thanks, they are currently a bit rough.